docs/outgoing-webhooks.md

```diff
@@ -0,0 +1,137 @@
+# Outgoing Webhooks
+
+These are a type of integration which post data to third party URLs through a HTTP POST request.
+To create an outgoing webhook, create an outgoing webhook bot from add new bot ui, with following details:
```

Comment:
I would reword this to be like:
To create an outgoing webhook, go to the "Add a new bot" tab in bot settings (?), and select the following options:

Point here is that we don't want to say "outgoing webhook" repetitively.